mod_modular_vanilla/hooks/skills/actives/strike_skill.nut

```diff
@@ -0,0 +1,21 @@
+::ModularVanilla.MH.hook("scripts/skills/actives/strike_skill", function(q) {
+	// Convert the vanilla method of "setting" certain fields to instead be incremental changes
+	q.onAfterUpdate = @() function( _properties )
+	{
+		if (this.m.ApplyAxeMastery)
+		{
+			if (_properties.IsSpecializedInAxes)
+			{
+				this.m.FatigueCostMult *= ::Const.Combat.WeaponSpecFatigueMult;
+			}
+		}
+		else if (_properties.IsSpecializedInPolearms)
+		{
+			this.m.FatigueCostMult *= ::Const.Combat.WeaponSpecFatigueMult;
+			if (this.m.ActionPointCost > 5)
+			{
+				this.m.ActionPointCost -= 1;
```

Comment:
Now Longaxes get a discount when you have Polearm Mastery. Because they also use the `strike_skill`
But in Vanilla that is not the case.